guizero/Painter.py

```diff
@@ -0,0 +1,26 @@
+from tkinter import Canvas, ALL
+from . import utilities as utils
+
+
+class Painter:
+    def __init__(self,master,width=100,height=100,background='white',grid=None,align=None):
```

Comment:
`background` should be persisted as an attribute, named `bg` and exposed as a property to be consistent with other widgets.